components/theme/app/AppHeader.vue

```diff
@@ -22,6 +22,12 @@
             :items="[link.items]"
             placement="bottom"
             mode="hover"
+            :dropdown-menu-class="
+              ('h-full', isHome ? 'bg-sky-darker text-white' : 'bg-cloud-surface dark:bg-sky-darker')
+            "
+            dropdown-class="w-max"
+            :open-delay="0"
+            :items-class="`py-1 grid grid-cols-${Math.round(link.items.length / 3)} gap-x-12`"
```

Comment:
I'm not really sure but the computation seems wrong.
You should divide by 4 and you should `ceil` instead of `round`: `Math.ceil(link.items.length / 4)`
Meaning that [from 1 to 4] -> [1 col], [from 5 to 8] -> [2 cols], etc.
Or I did miss something